public/wp-content/themes/glific/page-get-started.php

```diff
@@ -36,14 +36,31 @@
                         <?php endforeach;
                         endif;
                         if ($key === 2) : ?>
-                        <a href="<?php $section_one_data['contact_us_url']; ?>" class="d-block w-154 text-decoration-none bg-theme-primary text-white font-heebo-regular fz-18 px-10 py-4 leading-27 rounded-top-15 rounded-bottom-left-15">Contact Us</a>
+                            <a href="<?php $section_one_data['contact_us_url']; ?>" class="d-block w-154 text-decoration-none bg-theme-primary text-white font-heebo-regular fz-18 px-10 py-4 leading-27 rounded-top-15 rounded-bottom-left-15">Contact Us</a>
                         <?php endif; ?>
                         </div>
                     </div>
                 </div>
             <?php endforeach;
             endif; ?>
     </div>
+    <?php $section_two_data = get_field('section_two');
+    if (!empty($section_two_data)) : ?>
+    <div class="d-none d-md-flex flex-row w-xl-868 w-md-750 ml-xl-64 justify-content-between mb-md-26 mx-auto">
+    <?php foreach ($section_two_data as $key => $data) : ?>
+        <div class="d-flex flex-column w-xl-426 w-md-360 p-xl-10 p-md-9 bg-white get-started-content-box rounded-top-30 rounded-bottom-right-30">
+            <h2 class="fz-md-24 w-xl-345 w-md-310"><?php echo $data['title']; ?></h2>
+            <?php if (!empty($data['button_url']) && !empty($data['button_text'])) :?>
+            <div class="d-flex flex-row justify-content-center mt-xl-7 mt-md-8">
```

Comment:
Please use some formatting tools. These are easily available nowadays.
If you are using Visual Code, you can use `PHP IntelliSense` or `PHP Intelephense` or some other. Please do share if you found anything better.

Reply:
@Satendra-SR I am using `PHP Intelephense` from now onwards